collection_templates/collections__torch.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+    <identifier type="issn">2641-4775</identifier>
+    <identifier type="filename">torch_vol${volume}-no${number}</identifier>
+    <titleInfo supplied="yes">
+        <title>The Torchbearer, volume ${volume}, number ${number}</title>
```

Comment:
There is ${number} in the yml file. ${number} needs to be in both locations.

Reply:
I assume you meant "${issue}"?

Reply:
Thanks for this @DonRichards. Apologies for not taking enough time to look at this properly before committing. I've changed the variable so that it's number.